js/data.js

```diff
@@ -436,7 +491,7 @@ var data = {
         kennung: [ITSE, HCT, SAMT],
         lehrform: ["Seminar", "Projekt"],
         modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
-        semester: ss14,
+        semester: [ss14, ss15],
```

Comment:
In this case I was unsure, because the namees are very similar, and I thought, maybe it was meant to be the same. "HCI Project Seminar on Building Immersive Virtual Reality Games" vs. "HCI Project Seminar on Virtual Reality Gaming"